### ISSUE TITLE
speed 

### ISSUE THREAD

**Issue by MuyuXiaoxiang** [taolei87/sru#81]
I am sorry to disturb you.I don't know why the training time of LSTM and SRU is the same. The code is like this. Is there something that I should care? Thank you for yours reply!
`class net(nn.Module):
    def __init__(self, input_size, hidden_size, num_layers, batch_size):
        super(net, self).__init__()
        self.input_size = input_size
        self.hidden_size = hidden_size
        self.num_layers = num_layers
        #self.batch_size = batch_size
        #self.lstm = nn.LSTM(input_size, hidden_size, num_layers)
        self.lstm = SRU(input_size, hidden_size, num_layers)
        self.fc = nn.Linear(hidden_size, 1)

    net = net(input_size, hidden_size, layer_nums, batch_size)
    net = net.cuda()
    epoch = 2000
    criterion = nn.MSELoss()  # 损失函数
    optimizer = optim.SGD(net.parameters(), lr = 0.05)
    train_loss = []
    test_loss = []
    start_time = datetime.datetime.now()
    for i in range(epoch):
        for start, end in zip(range(0, len(x_train), batch_size), range(batch_size, len(x_train) + 1, batch_size)):
            x = torch.from_numpy(x_train[start:end])
            y = torch.from_numpy(y_train[start:end]).cuda()
            input = Variable(x).cuda()
            output = Variable(y).cuda()
            out = net(input)
            loss = criterion(out, y)
            optimizer.zero_grad()  # 梯度归零
            loss.backward()
            optimizer.step()`
